tests/test_visualization.py

```diff
@@ -0,0 +1,390 @@
+# Copyright (C) 2022 - 2025 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import sys
+
+import pytest
+
+# Windows CI runners don't have the capacity to hold fluent image.
+# Adding this makes sure that these tests only run in ubuntu CI.
+if sys.platform == "win32":
+    pytest.skip("Skipping this module on Windows", allow_module_level=True)
+else:
+    # These tests are failing in python 3.13. Investigate it along with
+    # "test_vector_object" and "test_surface_object" in "test_post.py".
+    if sys.version_info[:2] == (3, 13):
+        pytest.skip("Skipped on Python 3.13", allow_module_level=True)
+
+from unittest.mock import patch
+
+import ansys.fluent.core as pyfluent
+from ansys.fluent.core import examples
+import numpy as np
+import pytest
+import pyvista as pv
+
+from ansys.fluent.visualization import (
+    Contour,
+    GraphicsWindow,
+    Mesh,
+    Monitor,
+    Pathline,
+    Surface,
+    Vector,
+    XYPlot,
+    config,
+)
+from ansys.fluent.visualization.graphics.graphics_windows_manager import (
+    GraphicsWindow as TGraphicsWindow,
+)
+from ansys.fluent.visualization.graphics.pyvista.graphics_defns import Renderer
+from ansys.fluent.visualization.plotter.matplotlib.plotter_defns import (
+    Plotter as MatPlotter,
+)
+from ansys.fluent.visualization.plotter.plotter_windows_manager import (
+    PlotterWindow as TPlotterWindow,
+)
+from ansys.fluent.visualization.plotter.pyvista.plotter_defns import Plotter
+
+
+@pytest.fixture(scope="module")
+def new_solver_session():
+    solver = pyfluent.launch_fluent()
+    yield solver
+    solver.exit()
```

Comment:
We can add a `conftest.py` file to keep all fixtures in a single file.

Reply:
This is a module level one. That's why deleted the conftest.py. We might later introduce one if required with more tests coming-in, in future.